dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentQueryParams.java

```diff
@@ -72,8 +72,17 @@ class EnrollmentQueryParams {
   /** Selection mode for the specified organisation units. */
   private OrganisationUnitSelectionMode organisationUnitMode;
 
-  /** Program for which instances in the response must be enrolled in. */
-  private Program program;
+  /**
+   * Tracker program the tracked entity must be enrolled in. This should not be set when {@link
```

Comment:
Maybe not part of this PR but it would be good to enforce this rules at constructor/builder level so an invalid object cannot be created at all.